### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by anikitin35** [viankakrisna/vc-owl-carousel#2]
Please add to the project documentation for installing and using the module. 

Can we use this module to create slides in Visual Composer with grid and elements?

For example:
1. Create a slide in Visual Composer. 1/2 column is a text, 1/2 column is an image.
2. Create another slides as first.
3. Apply Owl Caruselle markup to slides created with Visual Composer.

**Comment by viankakrisna** [comment]
yup, that's the point of this vc module, but I haven't got time to maintain it. I create this to emulate the functionality of Advanced Carousel module of Ultimate Addons for Visual Composer (paid and using slick rather than owl carousel).
